games/2048/js/Tile.js

```diff
@@ -35,7 +35,7 @@ class Tile {
     if (this.value) {
       const textX = x + this.size / 2;
       const textY = y + this.size / 2;
-      this.ctx.font = `${this.size / 6}pt Arial`;
+      this.ctx.font = `${this.size / 6}pt Jura`;
```

Comment:
I think would be better move it to constans

Reply:
It is a dinamic value. Size can be changed

Reply:
Font family also can be changed?

Reply:
Got it!